03-visualization.Rmd

```diff
@@ -997,23 +1011,31 @@ Let's recap all five of the Five Named Graphs (5NG) \index{five named graphs} in
 ```{r viz-summary-table, echo=FALSE, message=FALSE, purl=FALSE}
 # The following Google Doc is published to CSV and loaded below using read_csv() below:
 # https://docs.google.com/spreadsheets/d/1vzqlFiT6qm5wzy_L_0nL7EWAd6jiUZmLSCFhDhztDSg/edit#gid=0
-summary_table_ch3 <-
-  "https://docs.google.com/spreadsheets/d/e/2PACX-1vRGaUW6EMIGPhg2V7CahoSdVi_JCcESFRYV5tov6bjcwOcn7DZDzfpZgrvjfFG6PV57gcJYIrwl_Q2c/pub?gid=0&single=true&output=csv" %>% 
-  read_csv(na = "") %>% 
-  rename_(" " = "X1") %>% 
-  kable(
-    caption = "Summary of 5NG", 
-    booktabs = TRUE
-  ) 
+
+if(!file.exists("rds/summary_table_ch3.rds")){
+  summary_table_ch3 <-
+    "https://docs.google.com/spreadsheets/d/e/2PACX-1vRGaUW6EMIGPhg2V7CahoSdVi_JCcESFRYV5tov6bjcwOcn7DZDzfpZgrvjfFG6PV57gcJYIrwl_Q2c/pub?gid=0&single=true&output=csv" %>% 
+    read_csv(na = "") %>% 
+    rename(` ` = `X1`)
+  write_rds(summary_table_ch3, "rds/summary_table_ch3.rds")
+} else {
+  summary_table_ch3 <- read_rds("rds/summary_table_ch3.rds")
+}
+
 if(knitr:::is_latex_output()){
   summary_table_ch3 %>% 
+    kable(
+      caption = "Summary of 5NG", 
+      booktabs = TRUE,
+      format = "latex"
+    ) %>% 
     kable_styling(font_size = ifelse(knitr:::is_latex_output(), 10, 16),
                   latex_options = c("HOLD_position")) %>%
     column_spec(3, width = "0.75in") %>%
     column_spec(4, width = "1.1in") %>%
     column_spec(5, width = "1.1in")
 } else {
-  summary_table_ch3
+  kable(summary_table_ch3, format = "html")
```

Comment:
Heads up, needed to add `caption` and `booktabs = TRUE` to html version too. Fixed in: c3a11a27bad30a1ab76b7c80aae3134c5922db63